assets/src/edit-story/components/fontPicker/test/fontPicker.js

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { act, fireEvent } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import FontPicker from '../';
+import { FontProvider } from '../../../app/font';
+import APIContext from '../../../app/api/context';
+import { renderWithTheme } from '../../../testUtils';
+import fontsListResponse from './fontsResponse';
```

Comment:
Instead of using this separate file, could we just use `/data/fonts.json`?

Reply:
But that's a variable file, right? That would make tests variable? This uses a static font list for test purposes - I think it's pretty on par with best practices. Yes, it does require we update the format in two places, but changing the format includes changing the test anyway.